DominicGBauer/information-access
# feat: added download button

# Pull Request

## Scope

Added in a button to download papers off the results card. 
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_backlogs/backlog/ECO5037S%20Front%20end%20Team/Backlog%20items/?workitem=132

## Work Done

1. Added a button that downloads the paper when clicked. 

## Steps to Test

1.  ```npm run start```
2. Do a search the produces results. 
3. Click on the download icon. 
4. Paper Should download


## Screenshot
![image](https://user-images.githubusercontent.com/80746660/135802492-6d79337d-dba6-4824-97d0-b5a741589e95.png)
